src/index.test.ts

```diff
@@ -150,7 +150,7 @@ describe('Running @erc725/erc725.js tests...', () => {
       assert.deepStrictEqual(dataArray, { 'NonExistingArray[]': [] });
     });
 
-    const e2eSchema: any = [
+    const legacyE2eSchema: any = [
```

Comment:
This test here is about checking whether or not the lib returns what it is supposed to return with legacy or not legacy mock.

If the schema itself is using legacy "keys" or not is not in the scope of this test.

We just check with one schema, see the results, validate or invalidate the tests. Using key A or key B is out of scope.

I therefore recommend to not duplicate the schema and not add `legacy` in the schema name.

Could you remove any occurrence of `legacy` in this test file and just keep the test